lib/recurly/request.php

```diff
@@ -36,13 +38,13 @@ public function getMethod(): string
     }
 
     /**
-     * Returns the path that the HTTP request was made to
+     * Returns the URL that the HTTP request was made to
      * 
-     * @return string The path
+     * @return string The URL
      */
-    public function getPath(): string
+    public function getUrl(): string
```

Comment:
Any chance of this being a breaking change?

Reply:
This `Request` class existed, but was never used